app/web_framework/cluster_base_handler.py

```diff
@@ -192,3 +192,12 @@ def _is_request_origin_allowed(self, request_origin):
         self._logger.debug('Origin "{}" did not match cors_allowed_origins_regex conf value of "{}". '
                            'Not setting Access-Control-Allow-Origin header.', request_origin, allowed_origins_regex)
         return False
+
+
+class Throws404Handler(ClusterBaseAPIHandler):
+    def initialize(self, route_node=None, **kwargs):
+        self._logger = log.get_logger(__name__)
+        super().initialize(route_node)
+
+    def get(self):
+        raise ItemNotFoundError
```

Comment:
This returns a response that looks like:

```json
{
    "error": "",
    "child_routes": {}
}
```

How do we want this to look / is there an proper way to customize this?